package.json

```diff
@@ -67,6 +67,7 @@
     "@dnd-kit/sortable": "^7.0.2",
     "@dnd-kit/utilities": "^3.2.1",
     "@openmrs/esm-framework": "next",
+    "@openmrs/esm-patient-common-lib": "next",
```

Comment:
This is necessary because of the recently added peer dependency (commons lib) as part of this [commit](https://github.com/openmrs/openmrs-form-engine-lib/commit/3fa8c75b3f5808d407df92821dab99e7b39b49e7).